helmfile.d/stacks/kured.yaml.gotmpl

```diff
@@ -17,8 +17,10 @@ templates:
       - values/podsecuritypolicies/common/kured.yaml.gotmpl
 
   kured-secret:
-    inherit: [ template: kured ]
     installed: {{ and (.Values | get "kured.enabled" false) (.Values | get "kured.notification.slack.enabled" false) }}
+    namespace: kured
+    labels:
+      app: kured
```

Comment:
Instead of removing the `inherit`, move the `installed` of the template onto both `kured-podsecuritypolicy` and `kured-main`, that is what was intended at least.
`installed` shouldn't be in a template unless it never needs to be overridden as Helmfile will merge over `false`.